packages/catalog-model/src/validation/CommonValidatorFunctions.ts

```diff
@@ -92,4 +92,35 @@ export class CommonValidatorFunctions {
       /^[a-z0-9]+(\-[a-z0-9]+)*$/.test(value)
     );
   }
+
+  /**
+   * Checks that the value is a valid URL.
+   *
+   * @param value The value to check
+   */
+  static isValidUrl(value: unknown): boolean {
+    if (typeof value !== 'string') {
+      return false;
+    }
+
+    try {
+      const url = new URL(value);
+      return !!url;
```

Comment:
This was to work around linting errors. Eslint doesn't like `new URL(value)` (no-new violation). Assigning it to a variable and returning true means the variable isn't used 😞

Reply:
I see. Anyway I think using the URL constructor like this to check was the right decision, doing it with a regex or so is just bound to be less robust.

Reply:
Oh and by the way - since this was a perfectly legitimate use of a bare `new`, it would be fine to just silence the lint error with a comment as well.